wordpress-mobile/WordPress-iOS
# Issues/track support identity usage

Bumps analytics to track usage of ActivityLog/Rewind and setting the support identity.

To test:
ActivityLog/Rewind:
Confirm the stats are bumped when viewing the list, when viewing a detail, and when starting a rewind.

Support Identity
From the Me > Help & Support screen, confirm the stat is bumped when the Contact Email form is viewed and when the dialog's OK button is tapped (but not the cancel button).
